composer.json

```diff
@@ -179,6 +179,7 @@
         "codeception/module-webdriver": "^3.2.1",
         "codeception/phpunit-wrapper": "^9.0.7",
         "friendsofphp/php-cs-fixer": "^3.3.1",
+        "php-mock/php-mock-phpunit": "^2.7",
```

Comment:
this should be also added to composer.json file in framework package

Reply:
Nice catch